src/main/java/uk/gov/pay/api/service/AgreementService.java

```diff
@@ -69,33 +72,45 @@ private AgreementLinks createLinksFromMandateResponse(MandateConnectorResponse m
         agreementLinks.addKnownLinksValueOf(mandate.getLinks());
         return agreementLinks;
     }
-    
-    private boolean isFound(Response connectorResponse) {
-        return connectorResponse.getStatus() == HttpStatus.SC_OK;
-    }
 
-    private boolean isCreated(Response connectorResponse) {
-        return connectorResponse.getStatus() == HttpStatus.SC_CREATED;
-    }
-
-    private Response createAgreement(Account account, CreateAgreementRequest createAgreementRequest) {
+    Response createMandate(Account account, MandateConnectorRequest mandateConnectorRequest) {
```

Comment:
is it deliberate that this method is neither private nor public?

Reply:
This is package protected for testing purposes, not needed to be `public`